app/controllers/parties_controller.rb

```diff
@@ -0,0 +1,37 @@
+class PartiesController < ApplicationController
+  def new
+    @id = params[:external_movie_id]
+    @title = params[:title]
+    @duration = params[:duration]
+    @party = Party.new
+  end
```

Comment:
Just noting here again after our convo that we should wait on this until my piece is merged in and we can maybe refactor to use objects instead of constantly passing params